### DISCUSSION TITLE
Modal not working

### DISCUSSION THREAD

**Discussion by mftaa** [saadeghi/daisyui#1917] [Q&A] [answered]
hi, i use daisyui 3.0.3 on my nextjs project. when i try use modal, copy from daisy ui component. i clicked open modal button and the modal not showing. my page just show a shadow, not the content of the modal should be. how could that happen?

**Comment by saadeghi** [comment]
Is the problem fixed?

**Reply by scott-gurney** [reply]
was there a fix? As I am having the same issue.

Using code without css works, but adding the css has same issues as OP

**Reply by scott-gurney** [reply]
i swapped to the legacy checkbox version and it is working

**Comment by mftaa** [comment, marked as answer]
Yes, thanks

Pada tanggal Sen, 5 Jun 2023 16.57, Pouya Saadeghi ***@***.***>
menulis:
